doc/programming_guide/asrc/overview.rst

```diff
@@ -0,0 +1,169 @@
+********
+Overview
+********
+
+This is the XCORE-VOICE Asynchronous Sampling Rate Converter (ASRC) example design.
+
+The example system implements a stereo |I2S| slave and a stereo adaptive UAC2.0 interface and exchanges data between the two interfaces.
```

Comment:
I'd love to do this! However, changing it just for this example will be confusing since we use the master/slave terminology all through the code and documentation. We need to change everywhere including the drivers if we decide to do this.